docs/dxp/7.x/en/site-building/site-appearance/site-appearance.md

```diff
@@ -12,46 +12,33 @@ The design and functionality of a Liferay site can be adjusted and customized th
 
 ## Master Page Templates
 
-Coming soon!
-<!-- TODO:
+[Master Page Templates](../creating-pages/defining-headers-and-footers/master-page-templates.md) allow you to define elements for all of your pages using Fragments, especially headers and footers. You can use them to conform any number of pages on your Site to the look and feel you design.
```

Comment:
I wanted to specifically point out this article, since it was largely just a placeholder before -- but if we're going to merge it, then it needs at least some barebones content in it, I think.

I sort of struggled to write this (as I felt I was repeating myself a lot, as far as "different ways to customize your Site"). If you have any thoughts as to how this is presented, or if you'd like to see something different as a starting point for this, please feel free to share them.

Reply:
I can think of cutting the text a little bit:

_"allow you to define common elements for all of your pages using Fragments, especially headers and footers. "_

Does that make sense?

Reply:
Yeah, that makes sense. I can go with that